web.test.tests/web.test.tests/Pages/HistoryPage.cs

```diff
@@ -0,0 +1,121 @@
+﻿using OpenQA.Selenium;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace web.test.tests.Pages
+{
+    class HistoryPage
+    {
+        private IWebDriver driver;
+
+        public static String ClearBtnId = "clear";
+
+        public HistoryPage(IWebDriver driver)
+        {
+            this.driver = driver;
+        }
+
+        public IWebElement Calculator => driver.FindElement(By.XPath("//div[text()='Calculator']"));
+        public IWebElement ClearBtn => driver.FindElement(By.Id(ClearBtnId));
+
+        public HistoryPage Clear()
+        {
+            ClearBtn.Submit();
+            return new HistoryPage(driver);
+        }
+
+        public DepositPage BackToDeposit()
+        {
+            Calculator.Click();
+            return new DepositPage(driver);
+        }
+
+        public int GetRowsNumber()
+        {
+            return driver.FindElements(By.XPath("//table[@id='history']/tr")).Count;
+        }
+
+        public IWebElement GetNRow(int row_number)
+        {
+            return driver.FindElement(By.XPath($"//table[@id='history']/tr[{row_number}]"));
+        }
+
+        public bool IsHistoryClean()
+        {
+            try
+            {
+                driver.FindElement(By.XPath($"//table[@id='history']/tr"));
+                return false;
+            } catch (NoSuchElementException e)
+            {
+                return true;
+            }
+        }
```

Comment:
FindElement will throw exception, if the element doesn't exist, but FindElement**s** thows no exceptions. So you may write it shorter:

```suggestion
        public bool IsHistoryClean => GetRowsNumber() == 0;
```

Reply:
Cool ))